system/lib/pthread/library_pthread_stub.c

```diff
@@ -407,3 +407,25 @@ void emscripten_thread_sleep(double msecs) {
     // Do nothing.
   }
 }
+
+int pthread_setconcurrency(int val)
+{
+  return 0;
+}
+
+int pthread_getconcurrency()
+{
+  return 0;
+}
+
+int pthread_attr_setscope(pthread_attr_t *a, int scope)
+{
+  switch (scope) {
+  case PTHREAD_SCOPE_SYSTEM:
+    return 0;
+  case PTHREAD_SCOPE_PROCESS:
+    return ENOTSUP;
+  default:
+    return EINVAL;
+  }
+}
```

Comment:
It looks like our musl's implementation (which is a stub) should work in both threaded and non-threaded builds so we can perhaps instead add `pthread_setconcurrency.c`, `pthread_getconcurrency.c` and `pthread_attr_setscope.c` to the list on line 105 of system_libs.py?

Reply:
That is a great point. Went through the musl pthread files and added the ones that look safe.